packages/cli/src/help/index.ts

```diff
@@ -0,0 +1,42 @@
+import { Help } from '@oclif/core'
+import { BaseCommand } from '@preevy/cli-common'
+
+export default class MyHelpClass extends Help {
```

Comment:
the name `MyHelpClass` doesn't reflect the functionality this class

Reply:
How about customHelpClass ?